src/apps/default.ts

```diff
@@ -21,5 +21,6 @@ export function defaultApp(
 
     res.render("probot.handlebars", pkg);
   });
-  router.get("/", (req, res, next) => res.redirect("/probot"));
+
+  router.get("/", (req, res) => res.redirect("/probot"));
```

Comment:
this basically doesnt work anymore, because @octokit/webhooks createNodeMiddleware already sets root path "/" so overwriting is not possible.

Reply:
`createNodeMiddleware` should take an argument to configure its base path. We should receive webhooks at `/api/github/webhooks`. It's okay to list that as a breaking change. It will align Probot with @octokit

Reply:
I concur, `createNodeMiddleware` accepts an argument to set the path, we simply set a default value

https://github.com/octokit/webhooks.js/blob/8ae4d3f509462098c6390ee78ca3b532196c4218/src/middleware/node/index.ts#L9

Reply:
done